src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -249,7 +249,7 @@ protected override void ProcessRecord()
                 }
                 else
                 {
-                    dateToUse = Date;
+                    dateToUse = DateTime.SpecifyKind(Date, DateTimeKind.Utc);
```

Comment:
Why do we need the change? It looks like unrelated the PR.

Reply:
I added a few comments about this earlier, I wasn't sure if it'd be better to open a new PR for this change, or bundle it together with this one. 

This was done to address issues with using the `-AsUTC` parameter, with a `DateTime` with `kind` set as `Unspecified`. Due to the way `ToUniversalTime()` is handling the local offset, the resulting `DateTime` is off.  

Do you reckon this change should be added in a different PR?

Reply:
Yes, it is not related the PR.

Reply:
Sounds good, I'll revert this change and open a new PR 😁

Reply:
I think it will a breaking change. So please open new issue to discuss.

Reply:
Will do! I'll get that typed up now

Reply:
Sorry AsUtc was added in Preview1 and did not be jet released - no breaking change.

Reply:
I've submitted a new issue regarding this to get a conversation going, and ensure I'm interpreting things correctly 😁